### ISSUE TITLE
Horizontal scrollbar on "3D Card Flip"

### ISSUE THREAD

**Issue by thomaspink** [GoogleChrome/ui-element-samples#46]
Wild horizontal scrollbar appears because of `text-indent: -10000px;` on the flip buttons.

**Possible Solution:**
Add overflow hidden to the .front and .back div

``` css
sc-card .front,
sc-card .back {
  backface-visibility: hidden;
  position: absolute;
  width: 100%;
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
  border-radius: 3px;
  overflow: hidden; /* <- here */
}
```

**Side effect:**
On certain (medium sized) displays, there are now no scrollbars. But if the card reaches its max size when animating and the cards gets now bigger than the viewport a vertical scrollbar appears (only for the short time the card is bigger than the viewport). This leads to a "jumping/jiggle effect" because the viewport is now narrower (width of scrollbar) than before.
**Possible Solution:**
Always show the horizontal scrollbar.

``` css
body {
  overflow-x: hidden;
  overflow-y: scroll;
}
```

**Comment by thomaspink** [comment]
Strange: The side effect seems to only appear on the first flip/animation ... confused :/

It also makes me wonder that transform causes scrollbars ... mhmm

**Comment by ramlmn** [comment]
Or you could just add `overflow: hidden;` to the `button` element only.

``` css
sc-card button {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: none;
  border: none;
  text-indent: -10000px;
  overflow: hidden; /* <- here */
}
```

**Comment by ramlmn** [comment]
This also causes the content to be scaled out on page load due to lack of `initial-scale` in mobile devices

![mobile](https://cloud.githubusercontent.com/assets/12368291/18034875/0ecb0772-6d66-11e6-852e-91e6532b1d1e.jpg)

**Comment by thomaspink** [comment]
closed - PR #47 

@ramlmn maybe open a new issue for the scale bug on mobile?
